### DISCUSSION TITLE
Prototypes

### DISCUSSION THREAD

**Discussion by prgmr-1** [codefori/vscode-ibmi#2167] [IBM i]
In RDi if you use the /copy, UIVP9011PR (for the prototype)  in the source you can use ILH. (record format definition)  and it brings up a popup of all the fields in that prototype.  Is there an equivalent in VSCode?

**Comment by prgmr-1** [comment]
shouldn't /copy "bring in" what ever prototype that is there?

**Reply by worksofliam** [reply]
If you hover over the `/copy` or `/include` statement, it will say whether the member was 'found' or 'not found'.

**Reply by prgmr-1** [reply]
shows found but when I am in a subroutine for a file access prototype and type in say IVL followed by the period (invoice log prototype) it doesn't popup a list of fields in that file (does in RDi).

**Reply by worksofliam** [reply]
@prgmr-1 Screenshots are welcome, or code to recreate. Thanks

**Reply by prgmr-1** [reply]
can do both but will need help with how.
